model/validate.php

```diff
@@ -230,4 +233,53 @@ function validateFileUpload($file, $newName)
         }
         return $isValid;
     }
+
+    /**
+     * Returns true if the item data is valid
+     *
+     * @param string $itemName The name of the item
+     * @return bool Returns true if the item data is valid
+     * @author Keller Flint
+     */
+    function validateItem($itemName)
+    {
+        $isValid = true;
+        if (empty(trim($itemName))) {
+            $isValid = false;
+            $this->_errors["itemName"] = "Item name may not be empty";
+        }
+        if (strlen($itemName) > 255) {
+            $isValid = false;
+            $this->_errors["itemName"] = "Item name may not be greater than 255 characters";
+        }
+        return $isValid;
+    }
+
+    /**
+     * Returns true if the state data is valid
+     *
+     * @param int $stateId The id of the state
+     * @param string $stateName The name of the state
+     * @param string $stateText The description of the state
+     * @return bool Returns true if the state data is valid
+     * @author Keller Flint
+     */
+    function validateState($stateId, $stateName, $stateText)
+    {
+        $isValid = true;
+        if (empty(trim($stateName))) {
+            $this->_errors["stateName" . $stateId] = "Name may not be empty";
+            $isValid = false;
+        }
+        if (strlen($stateName) > 255) {
+            $isValid = false;
+            $this->_errors["stateName" . $stateId] = "Name may not be greater than 255 characters";
+        }
+
+        if (strlen($stateText) > 50) {
```

Comment:
Checking for greater than 50 characters, but error states 5000. Is this accurate?

Reply:
oops no i put that in for testing! Good catch!